includes/types.h

```diff
@@ -17,5 +45,12 @@ typedef enum 	e_error_codes
 	WRONG_ARGC,
 }				t_error;
 
+typedef struct 	s_minishell
+{
+	char		*prompt;

```

Comment:
Why is prompt part of the minishell struct? It's something we decide, so I think we can just have
#define PROMPT '> '

Reply:
ok !